### DISCUSSION TITLE
[9 gennaio 2023] Complessità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#3] [Complessità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/fef74a00-692c-42d9-b8c8-82a5169a780d)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/4a46402f-422b-4d7d-8657-de14e5630f9a)

-----

Il secondo punto, invece, è un diretto corollario del Teorema di Savitch, il quale può essere trovato sul libro di testo:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/89f1c832-fb5b-4938-861e-6ee69afc62e2)

**Comment by daromacarlo** [comment]
<img width="1620" height="537" alt="Immagine 2026-02-02 182217" src="https://github.com/user-attachments/assets/2905d0ce-701f-4f24-80fc-971b980f3796" />
<img width="1617" height="758" alt="Immagine 2026-02-02 182238" src="https://github.com/user-attachments/assets/eccc8183-00d7-4b65-8a7d-dd91ee0c3fa7" />
<img width="1615" height="265" alt="Immagine 2026-02-02 182253" src="https://github.com/user-attachments/assets/5e5b90cc-eeb5-4527-9375-8c56813f7848" />
<img width="1618" height="650" alt="Immagine 2026-02-02 182306" src="https://github.com/user-attachments/assets/bdca53b1-54dc-4b05-bf52-1011421c4d16" />
<img width="1619" height="231" alt="Immagine 2026-02-02 182330" src="https://github.com/user-attachments/assets/d26b0035-a05a-4363-bac4-80254cc6d690" />
